apps/spotlight/src/layouts/Layout.astro

```diff
@@ -49,10 +49,12 @@ const context = getAppContext();
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
+    <script is:inline src={`${context.uswdsRoot}js/uswds-init.min.js`}></script>  
     <script src="../lib/initialize.ts"></script>
     <title>{title}</title>
   </head>
   <body id="top">
+    <script is:inline defer src={`${context.uswdsRoot}js/uswds.min.js`}></script>
```

Comment:
```suggestion
    <script defer src={`${context.uswdsRoot}js/uswds.min.js`}></script>
```

Reply:
Hi Dan. I mentioned this in another comment, but it might have been overlooked. The reason both scripts include the is:inline directive, and why I didn’t remove it, is that it’s necessary for the scripts to load correctly. According to the [Astro documentation](https://docs.astro.build/en/guides/client-side-scripts/#load-external-scripts), this directive is required for loading scripts that are outside the Astro src folder. I confirmed this by testing the scripts without the directive, and they didn’t appear in the DOM at all.